internal/fileutil/fileutil.go

```diff
@@ -51,6 +51,14 @@ func Exists(path string) bool {
 	return err == nil
 }
 
+func IsDirEmpty(path string) bool {
+	entries, err := os.ReadDir(path)
+	if err != nil {
+		return false
```

Comment:
I think returning false on error is a bit confusing (is it not empty if it doesn't exist?)

`IsDirEmpty` could return the error as well and `dirExistsAndIsNotEmpty` handles the error and returns a single bool